packages/cookies-bundle/src/DependencyInjection/Configuration.php

```diff
@@ -24,19 +24,18 @@ public function getConfigTreeBuilder(): TreeBuilder
         $rootNode = $treeBuilder->getRootNode();
 
         $rootNode->children()
-                ->arrayNode('cookies')
-                    ->arrayPrototype()->children()
-                        ->scalarNode('name')->end()
+            ->arrayNode('cookies')
+                ->arrayPrototype()->children()
+                    ->arrayNode('name')->children()
                         ->booleanNode('has_description')->end()
-                        ->arrayPrototype()->children()
-                                ->scalarNode('name')->end()
-                                ->scalarNode('domain')->end()
+                        ->arrayNode('cookies')->children()
+                            ->scalarNode('name')->end()
+                            ->scalarNode('domain')->end()
                         ->end()
-                        ->end()
-                    ->end()
                     ->end()
                 ->end()
             ->end()
+        ->end()
         ;
 
         return $treeBuilder;
```

Comment:
Estoy teniendo problemas para sacar esta config, cuando lanzo el test me lanza un: 
Unrecognized options "0, 1" under "runroom_cookies.cookies.mandatory_cookies". Available option is "name". 
En el configuration.yaml, ¿"- name" se considera un array? Ahora mismo estoy entendiendo que por ejemplo, dentro de mandatory_cookies,  "symfony" es un array y "consent" es otro y de ahí el 0, 1? Pero no sé si viene de ahí el error.